examples/wearable/UIComponents/contents/navigationelements/index.html

```diff
@@ -50,6 +50,11 @@ <h2 class="ui-title">
 						Assist Panel
 					</a>
 				</li>
+				<li>
+					<a href="gototopbutton/index.html">
```

Comment:
go-to-top-button ?

Reply:
@grzegorz-czajkowski good question.
Not sure if kebap-case style refers also to directories. We need to find consensus

Reply:
ok, decided to change this